### PR TITLE
feat: Italian Translation

### LINKED ISSUE #833: feat: Italian Transaltion
### Is your feature request related to a problem? Please describe.

Generally the more docs languages you have, the more accesible it is to developers. 

### Describe the solution you'd like to see

Translating the docs to Italian

### Describe alternate solutions

I don't know, maybe Google Translate? But that would cause some confusion and just be a hassle

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by micziz**
Closes #833 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Add Italian translation

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/config.ts [hunk 1]
```diff
@@ -92,6 +121,11 @@ export const SIDEBAR_HEADER_MAP: Record<
   Exclude<KnownLanguageCode, "en">,
   Record<keyof Sidebar["en"], string>
 > = {
+  it: {
+    "Crate T3 App": "Create T3 App",
```
Comment (c-ehrlich) [inline www/src/config.ts:125]: typo here? ("Crate")

File: www/src/config.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export type Frontmatter = {
 
 export const KNOWN_LANGUAGES = {
   en: "🇺🇸 English",
+  it: "🇮🇹 Italian"
```
Comment (c-ehrlich) [inline www/src/config.ts:29]: Should be "Italiano" I think? We are using the name of the language, in that language.

File: www/src/config.ts (hunk 1 above)
Comment (micziz) [reply www/src/config.ts:125]: Oh right

File: www/src/config.ts (hunk 2 above)
Comment (micziz) [reply www/src/config.ts:29]: Oh ok missunderstud!

**Comment by micziz** [comment]
I will add the finishing touches and I think it's ready!

**Comment by juliusmarminge** [comment]
Please check out the new [TRANSLATIONS.md](https://github.com/t3-oss/create-t3-app/blob/next/www/TRANSLATIONS.md) and the notes in [config.ts](https://github.com/t3-oss/create-t3-app/blob/next/www/src/config.ts) and configure the sidebar for your language.

**Pull request closed without merging**